cross-modal-search/README.md

```diff
@@ -1,4 +1,4 @@
-# Build a CrossModal Search System to look for Images from Captions
+# Build a Cross-modal Search System to Look for Images from Captions
```

Comment:
Still not Title Case, every word should be in caps

Reply:
https://github.com/jina-ai/examples/pull/614/commits/0a91445beb1103999ebadd2c931ef30843e03a5e